common4j/src/main/com/microsoft/identity/common/java/commands/parameters/CommandParameters.java

```diff
@@ -79,11 +81,27 @@ public class CommandParameters {
     @Expose()
     private boolean powerOptCheckEnabled;
 
+    //Marking as final excludes from lombok generated builder
+    @Expose()
+    private final String applicationIdentifier;
```

Comment:
nit: do we need this?

Reply:
Nope.  I'll double check that it's not actually be used somewhere before I remove it.

Reply:
Confirmed.  Not needed.  Removed.